Cql/CodeGeneration.NET/CSharpSourceCodeWriter.cs

```diff
@@ -200,35 +200,12 @@ private void writeClass(DefinitionDictionary<LambdaExpression> definitions,
             if (PartialClass)
                 writer.WriteLine(indentLevel, $"partial class {className}");
             else
-                writer.WriteLine(indentLevel, $"public class {className}");
+                writer.WriteLine(indentLevel, $"public static class {className}");

```

Comment:
Line 201 must be `public partial static`

Reply:
added